ush/parsing_namelists_FV3.sh

```diff
@@ -106,15 +127,15 @@ cat > input.nml <<EOF
   p_fac = 0.1
   k_split = $k_split
   n_split = $n_split
-  nwat = ${nwat:-2}
+  nwat = ${nwat:-2}		! CROW configured
```

Comment:
Is there a reason for adding the comment "! CROW configured" to these variables?  (I mostly ask because this shows up in the differences between old and new).

Reply:
Nevermind, I see below this is distinguishing which is in config.fcst versus CROW and that can be helpful information.